oshi-core/src/main/java/oshi/software/os/unix/aix/AixOSProcess.java

```diff
@@ -349,17 +356,19 @@ static State getStateFromOutput(char stateValue) {
         State state;
         switch (stateValue) {
         case 'O':
+            state = INVALID;
+            break;
+        case 'A':
             state = RUNNING;
             break;
-        case 'S':
-            state = SLEEPING;
+        case 'I':
+            state = NEW;
             break;
-        case 'R':
         case 'W':
-            state = WAITING;
+            state = WAITING; //need confirmation on this
             break;
         case 'Z':
-            state = ZOMBIE;
+            state = CANCELED;
```

Comment:
I think this might be the same thing as Zombie.  [From this page](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxa500/ps.htm):
> Z
> Canceled and parent has not performed wait (Z for zombie).

Reply:
ok i think i need to read both [man ps](https://www.ibm.com/support/knowledgecenter/ssw_aix_72/p_commands/ps.html) and [ps returns](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxa500/ps.htm). Thanks for pointing out